packages/core-flows/src/definition/cart/workflows/complete-cart.ts

```diff
@@ -0,0 +1,19 @@
+import { CompleteCartWorkflowInputDTO } from "@medusajs/types"
+import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
+
+/*
+  - [] Create Tax Lines
```

Comment:
I guess we should do some validation beforehand, that the prices of the products/cart haven't changed, the shipping method is still available, etc.

Reply:
@riqwan sounds good, it's fine to get the boilerplate in place anyway 👍